src/it/scala/org/apache/spark/sql/SFDataFrameWindowFunctionsSuite.scala

```diff
@@ -0,0 +1,66 @@
+package org.apache.spark.sql
+
+import org.apache.spark.sql.snowflake.{SFQueryTest, SFTestData, SFTestSessionBase}
+
+class SFDataFrameWindowFunctionsSuite
+    extends DataFrameWindowFunctionsSuite
+    with SFTestSessionBase
+    with SFQueryTest
+    with SFTestData {
+
+  override def spark: SparkSession = getSnowflakeSession()
+
+  override protected def blackList: Seq[String] =
+    Seq(
```

Comment:
Why would you blacklist *anything* ? Unlike TS, the idea here is for us to push what's possible and execute the rest in Spark, correct ?

Reply:
Yeah, as I mentioned, I'd like to remove from the blacklist tests that work here:
https://snowflakecomputing.atlassian.net/browse/SNOW-201512

I still see some failures when I try to run the whole suite, and there's a large overlap between what fails and what is on this SnowPark-inherited blacklist, so they'll need to be investigated.

For now, I see this blacklist as still having a use: to make sure there aren't any failures in our integration tests while we patch things up.